src/js/mozilla-dnthelper.js

```diff
@@ -0,0 +1,65 @@
+/* eslint-disable no-octal-escape */
```

Comment:
This code has been updated from the kuma version to make eslint happier, plus also to pin the `Mozilla` object on window, to make it reachable from the template. 

Please shout if you think this isn't the right way to go